app/src/main/java/org/fossasia/openevent/activities/SpeakersActivity.java

```diff
@@ -0,0 +1,23 @@
+package org.fossasia.openevent.activities;
+
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.widget.TextView;
+
+import org.fossasia.openevent.R;
+
+/**
+ * Created by MananWason on 30-06-2015.
+ */
+public class SpeakersActivity extends AppCompatActivity {
+    private String text;
```

Comment:
Can we be a more descriptive here? Also what does text mean exactly? can we capture this information in the Database and load it using a loader?